### ISSUE TITLE
Prerequisite - dateutil

### ISSUE THREAD

**Issue by damies13** [damies13/har2rf-req#1]
```
pip install python-dateutil
```

**Comment by damies13** [comment]
I don;t remember what I raised this issue for? I'm sure it'll come back to me at some point

**Comment by kstine** [comment]
Your tool requires installing python-dateutil

**Comment by damies13** [comment]
yes, probably the issue was to remind me to do something like put it as a dependency in in a `setup.py` or `requirements.txt` file.

**Comment by damies13** [comment]
Are you using this tool? I never even announced it, but if your finding it useful i should put more time into it.
